### PR TITLE
More env span related cleanups

### PR DESCRIPTION
**Pull request by pattyshack**

### CODE REVIEW

File: fvm/contractFunctionInvoker.go [hunk 1]
```diff
@@ -41,17 +40,13 @@ func NewContractFunctionInvoker(
...
 }
 
-func (i *ContractFunctionInvoker) Invoke(env Environment, parentTraceSpan otelTrace.Span) (cadence.Value, error) {
+func (i *ContractFunctionInvoker) Invoke(envCtx *EnvContext, env Environment) (cadence.Value, error) {
 
-	ctx := env.Context()
-	if ctx.Tracer != nil && parentTraceSpan != nil {
-		span := ctx.Tracer.StartSpanFromParent(parentTraceSpan, trace.FVMInvokeContractFunction)
-		span.SetAttributes(i.logSpanAttrs...)
+	span := envCtx.StartSpanFromRoot(trace.FVMInvokeContractFunction)
```
Comment (Tonix517) [inline fvm/contractFunctionInvoker.go:45]: could `envCtx` possibly be `nil` here? is it necessary to check nil for `envCtx` here? same question for `env`

**Review by Tonix517 — APPROVED** [review]

**Review by janezpodhostnik — APPROVED** [review]

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors retry

**Comment by pattyshack** [comment]
bors retry

**Comment by pattyshack** [comment]
bors retry

**Comment by pattyshack** [comment]
bors retry

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**